from dataview.api.log import Log
from dataview.api.exception import Exception

log = Log()
exception = Exception()
